tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowsersecuritychange_event.html

```diff
@@ -0,0 +1,49 @@
+<head>
+<script src="/resources/testharness.js"></script>
```

Comment:
Let's give this test a `<title>` and update the ini file to use it instead of `Untitled`.